exo-go/test_helpers/shared_feature_context.go

```diff
@@ -131,7 +131,7 @@ func SharedFeatureContext(s *godog.Suite) {
 
 	s.Step(`^it prints "([^"]*)" in the terminal$`, func(text string) error {
 		if childCmdPlus != nil {
-			return childCmdPlus.WaitForText(text, time.Minute)
+			return childCmdPlus.WaitForText(text, time.Minute*2)
```

Comment:
We don't know the order in which the tests are run (it is random I believe). So when we wait for text [here](https://github.com/Originate/exosphere/blob/3cc763de668a7096a772303cc3cf0f301d862609/exo-go/features/test/test.feature#L11), we might end up giving up on waiting because `tweets-service` gets run before `users-service`.